Data/Sys/GameSettings/GSWD64.ini

```diff
@@ -0,0 +1,29 @@
+# GSWD64 - Star Wars: Rogue Leader
+
+[Core]
+# Values set here will override the main dolphin settings.
+MMU = 1
+FastDiscSpeed = 1
```

Comment:
If this gets merged first, it's okay to leave as-is; just wanted to make sure you were aware of the interaction.